atlantis-ci/variables.tf

```diff
@@ -26,11 +26,6 @@ variable "allowed_repositories" {
   description = "List of repositories using Atlantis"
 }
 
-variable "github_app_id" {

```

Comment:
These variables are actually in use, see my comment above.

Reply:
done